src/web/browser/App.tsx

```diff
@@ -15,6 +15,7 @@ import { SlotBodyEnd } from '@frontend/web/components/SlotBodyEnd';
 import { Links } from '@frontend/web/components/Links';
 import { SubNav } from '@frontend/web/components/SubNav/SubNav';
 import { CommentsLayout } from '@frontend/web/components/CommentsLayout';
+import { incrementWeeklyArticleCount } from '@guardian/slot-machine-client';
```

Comment:
Is this name tied to frontend? Seems like the action here is to increment the article count, and whether it is a weekly count or not is dependent on how the data might be read later, not how it is written

Reply:
Good point, but that function does save the counter in a format that's "week-centric", so this is the most descriptive name for what it does - it happens to be the same used by Frontend which is a further convenience. We can rename the function but in this case there's no point as this is already descriptive.
I could have provided a link so you can actually see what that function does, here is the other PR - https://github.com/guardian/slot-machine-client/pull/12/files (search for the function name). @oliverlloyd

Reply:
Fair enough, sounds like you've got it covered. Thanks for the explanation